src/configure/templateInputHelper/InputControlProvider.ts

```diff
@@ -60,9 +60,12 @@ export class InputControlProvider {
             let dependentInputControlArray = this._getInputDependencyArray(inputControl, [properties[element]], false);
             let dependentClientInputMap = this._getClientDependencyMap(inputControl, [properties[element]]);
             let newValue = this._computeMustacheValue(properties[element], dependentInputControlArray, dependentClientInputMap);
+            if (!Object.keys(newValue).length) {
+                newValue = properties[element];
+            }
             inputControl.updateInputDescriptorProperty(key, newValue);
             if (key === constants.inputModeProperty) {
-                let updatedControlType = InputControlUtility.getInputControlType(InputMode[newValue]);
+                let updatedControlType = InputControlUtility.getInputControlType(parseInt(newValue.toString()));
```

Comment:
why this change?

Reply:
The InputMode in template can be an integer or string. Example, if it's a override scenario, it will be treated as a string and direct inputMode property comes as a number. To handle both the scenarios, this is done.